app/Jobs/ImportRockTheVotePosts.php

```diff
@@ -26,12 +26,10 @@ class ImportRockTheVotePosts implements ShouldQueue
     protected $filepath;
 
     /**
-     * The path to the stored csv.
+     * The total records in the stored csv.
```

Comment:
is this a count of the total records or is it the records themselves?

Reply:
@katiecrane great q! This is [the count](https://github.com/DoSomething/chompy/blob/7b32036b779ce702f50076bc1e60c8c87c75554f/app/Traits/ImportToRogue.php#L35) of the total records. Updated this comment to make it more clear!